Tests/OpusTests/OpusDecoderTests.swift

```diff
@@ -15,4 +15,24 @@ final class OpusDecoderTests: XCTestCase {
 			}
 		}
 	}
+
+
```

Comment:
SwiftFormat complained about this extra blank line: https://github.com/alta/swift-opus/runs/4634584911?check_suite_focus=true

Can you run `swiftformat` or delete this line?

Reply:
done